spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/procedures/SnapshotTableProcedure.java

```diff
@@ -93,10 +94,20 @@ public InternalRow[] call(InternalRow args) {
               });
     }
 
+    int parallelism;
+    if (!args.isNullAt(4)) {
+      parallelism = args.getInt(4);
+    } else {
+      parallelism = 1;
+    }
```

Comment:
Nit: I think it's a bit cleaner to use ternary here

`int parallelism = args.getInt(4) != null ? args.getInt(4) : 1`

Reply:
Thanks for your advice, I will fix this.

Reply:
> Nit: I think it's a bit cleaner to use ternary here
> 
> `int parallelism = args.getInt(4) != null ? args.getInt(4) : 1`

Thanks, I have fixed.